pkgs/by-name/ic/icann-rdap/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  perl,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "icann-rdap";
+  version = "0.0.22";
+
+  src = fetchFromGitHub {
+    owner = "icann";
+    repo = "icann-rdap";
+    rev = "v${version}";
+    hash = "sha256-J8uvFTjFY7YrohxlD7UwzrQUUoHHEb5mzZoQ64XUQHY=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-2R6GrcuksEOk8GiVkFhMljS12V2n0J1rCw9MCOtwMjA=";
+
+  nativeBuildInputs = [ perl ];
+
+  meta = with lib; {
+    description = "Official ICANN RDAP tools, containing cli client and server";
+    mainProgram = "rdap";
+    homepage = "https://github.com/icann/icann-rdap";
+    license = licenses.mit;
```

The repo has two licenses listed, MIT and Apache.
```suggestion
    homepage = "https://github.com/icann/icann-rdap";
    license = with lib.licenses; [ mit asl20 ];
```

---

fixed, nice catch!